dist/floating-label/floating-label.css

```diff
@@ -13,11 +13,10 @@ label.floating-label__label {
   display: inline-block;
   left: 16px;
   overflow: hidden;
-  padding-top: 3px;
   pointer-events: none;
   position: absolute;
   text-overflow: ellipsis;
-  top: -3px;
+  top: 0;
```

Comment:
The peek-through is better but still not as good as it might be. On my local, retaining the `-3px` for top actually yields even more coverage. It does move the label up 3px though but maybe that's better?

<img width="194" alt="image" src="https://user-images.githubusercontent.com/1675667/181361032-ecdf4bdf-3f21-414e-8e07-9eb1a4c52c71.png">

In general (not for this issue), it looks like we may need to restructure the markup a bit.

Reply:
The issue is this is common for textarea and select
The usecase for select is much more common than textarea. Plus the select focus is broken without this. 
If we want to fix both we need another modifier. 
Im fine with the bleed through for now on textarea. I want select to look better.

Reply:
Ok